docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2'
+
+volumes:
+  pg-data:
+
+services:
+  jeeves:
+    build: .
+    depends_on:
+      - db
+      - irc
+    environment:
```

Comment:
i really only preferred config files because you don't need to rewrite the run/launch scripts to run the bot

Reply:
sure. env vars are good for docker, config files are good for npm run.